### PR TITLE
Simplify diagnostic squiggling code

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
We were passing constant values through in certain places.

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
@dibarbet ptal.

File: global.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "sdk": {
     "version": "8.0.100-preview.7.23376.3",
     "allowPrerelease": false,
-    "rollForward": "latestPatch"
+    "rollForward": "disable"
```
Comment (genlu) [inline global.json:5]: is this change intentional?

Comment (CyrusNajmabadi) [reply global.json:5]: yes.  @jasonmalinowski for more context.  but this unblocks people being able to actually develop with roslyn locally :)

**Review by akhera99 — APPROVED** [review]
